app/src/main/res/drawable/ic_change_novideo.xml

```diff
@@ -0,0 +1,13 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
In the latest rebase you are missing to remove the following files in your last commit. Before the rebase you removed the `rc3` flavor files which was fine. Now you can remove the same `jev2022` flavor files (`ic_change_novideo.png`, `ic_change_video.png`, `ic_novideo.png`, `ic_without_video_recording.png`).

Reply:
Done in next Push